bazel/envoy_mobile_copts.bzl

```diff
@@ -1,8 +1,9 @@
 # DO NOT LOAD THIS FILE. Load envoy_build_system.bzl instead.
-load(":envoy_select.bzl", "envoy_select_admin_functionality", "envoy_select_enable_http3", "envoy_select_envoy_mobile_request_compression")
+load(":envoy_select.bzl", "envoy_select_admin_functionality", "envoy_select_enable_http3", "envoy_select_envoy_mobile_listener", "envoy_select_envoy_mobile_request_compression")
 
 # Compute the copts needed for Envoy Mobile libraries that don't use Envoy's main library wrappers.
 def envoy_mobile_copts(repository):
     return envoy_select_admin_functionality(["-DENVOY_ADMIN_FUNCTIONALITY"], repository) + \
            envoy_select_enable_http3(["-DENVOY_ENABLE_QUIC"], repository) + \
+           envoy_select_envoy_mobile_listener(["-DENVOY_MOBILE_ENABLE_LISTENER"], repository) + \
```

Comment:
This is fine here, but it could be in the `envoy_copts` macro since we don't need this defined for `swift_library`, `objc_library` or the JNI `cc_library`s.